README.md

```diff
@@ -12,7 +12,7 @@ Before using dbtools you'll need to install pydbtools v2.0.1 or higher. Do this
 
```

Comment:
I can't comment on the actual line, but the pydbtools version needs a bump on line 11. ☝️